docs/source/upgrading3.rst

```diff
@@ -74,8 +74,10 @@ Nonfunctional Options
 +++++++++++++++++++++
 
 ``tartufo`` release 3 uses
+
 .. _pygit2: https://pygit2.readthedocs.io/en/latest/
 instead of
+
```

Comment:
This should follow the convention of the rest of the documentation, and split the presentation of links from their definition. So here you would have:

```rst
``tartufo`` release 3 uses `pygit2`_ instead of `GitPython`_ to access git repositories.
```

Then at the very bottom of the file you would have

```rst
.. _pygit2: https://pygit2.readthedocs.io/en/latest/
.. _GitPython: https://gitpython.readthedocs.io/en/stable/
```

For more examples, see all of the pre-existing documentation. 😄